README.md

```diff
@@ -236,479 +236,900 @@ The base class sets defaults used by other defined types, such as `tomcat::insta
+
+Valid options: `true` and `false`.
+
+Default value: `true`.
 
 #####`manage_user`
 
-Determines whether defines should default to creating the specified user, if it doesn't exist. Uses Puppet's native [`user` resource type](https://docs.puppetlabs.com/references/latest/type.html#user) with default parameters. Valid options: 'true' and 'false'. Default: 'true'.
+Determines whether defines should default to creating the specified user, if it doesn't exist. Uses Puppet's native [`user` resource type](https://docs.puppetlabs.com/references/latest/type.html#user) with default parameters.
+
+Valid options: `true` and `false`.
+
+Default value: `true`.
 
 ##### `manage_base`
-Specifies the default value of `manage_base` for all `tomcat::install` instances. Default: `true`
+Specifies the default value of `manage_base` for all `tomcat::install` instances.
+
+Default value: `true`
 
 ##### `manage_home`
-Specifies the default value of `manage_home` for all `tomcat::instance` instances. Default: `true`
+Specifies the default value of `manage_home` for all `tomcat::instance` instances.
+
+Default value: `true`
 
 ##### `manage_properties`
-Specifies the default value of `manage_properties` for all `tomcat::instance` instances. Default: `true`
+Specifies the default value of `manage_properties` for all `tomcat::instance` instances.
+
+Default value: `true`
 
 #####`purge_connectors`
 
-Specifies whether to purge any unmanaged Connector elements that match defined protocol but have a different port from the configuration file by default. Valid options: 'true' and 'false'. Default: 'false'.
+Specifies whether to purge any unmanaged Connector elements that match defined protocol but have a different port from the configuration file by default.
+
+Valid options: `true` and `false`.
+
+Default value: `false`.
 
 #####`purge_realms`
 
-Specifies whether to purge any unmanaged realm elements from the configuration file by default. Valid options: 'true' and 'false'. Default: 'false'.  If two realms are defined for a specific server config only use purge\_realms for the first realm and ensure the realms enforce a strict order between each other.
+Specifies whether to purge any unmanaged realm elements from the configuration file by default.
+
+Valid options: `true` and `false`.
+
+Default value: `false`.  If two realms are defined for a specific server config only use purge_realms for the first realm and ensure the realms enforce a strict order between each other.
 
 #####`user`
 
-Specifies a default user to run Tomcat as. Valid options: a string containing a valid username. Default: 'tomcat'.
+Specifies a default user to run Tomcat as.
+
+Valid options: a string containing a valid username.
+
+Default value: `tomcat`.
 
 #### tomcat::config::properties::property
 Specifies an additional entry for the catalina.properties file a given catalina base.
```

Comment:
needs a space in between, and catalina.properties should be in backticks (filename).